.travis.yml

```diff
@@ -1,9 +1,10 @@
 language: go
 
 go:
-  - 1.7
+  - 1.7.x
```

Comment:
Out of curiosity, what does this do? Does it use the latest 1.7.x?

This should be clarified in the docs, as `a version with x in place of the minor or patch level` doesn't mean that to me.